config/jointcal.py

```diff
@@ -0,0 +1,22 @@
+import os.path
+
+# `load()` appends to the filterMaps: we need them to be empty, so that
+# only the specified filter mappings are used.
+config.photometryRefObjLoader.filterMap = {}
+filterMapFile = os.path.join(os.path.dirname(__file__), "filterMap.py")
+config.photometryRefObjLoader.load(filterMapFile)
+# We have PS1 colorterms
+config.applyColorTerms = True
+config.colorterms.load(os.path.join(os.path.dirname(__file__), "colorterms.py"))
+
+# This value copied from obs_subaru without modification. Original comment:
+# HSC needs a higher order polynomial to track the steepness of the optical
+# distortions along the edge of the field. Emperically, this provides a
+# measurably better fit than the default order=5.
+config.astrometryVisitOrder = 7
+
+# This value copied from obs_subaru without modification. Original comment:
+# For the deep fields with hundreds of visits, outlier rejection may take many
+# weeks of runtime. By stopping outlier rejection when it ceases to have a
+# significant effect on the model, we can bring compute time down to a few days.
+config.astrometryOutlierRelativeTolerance = 0.002
```

Comment:
Lets leave this alone until we run jointcal on enough visits where the runtime matters: jointcal on DECam should be significantly faster than on HSC (less stars/visit and less detectors).